runsc/fsgofer/fsgofer.go

```diff
@@ -955,14 +955,14 @@ func (l *localFile) Readdir(offset uint64, count uint32) ([]p9.Dirent, error) {
 }
 
 func (l *localFile) readDirent(f int, offset uint64, count uint32, skip uint64) ([]p9.Dirent, error) {
+	var dirents []p9.Dirent
+
 	// Limit 'count' to cap the slice size that is returned.
-	const maxCount = 100000
+	const maxCount = 10000
```

Comment:
Why change this?

Reply:
Oops, I forget to drop this change. Will resubmit.